mods/gen/rules/vehicles.yaml

```diff
@@ -2477,7 +2513,7 @@ vehicle.tomahawk_launcher:
 		VoiceSet: TomahawkLauncherVoice
 	ActorStatValues:
 		Damage: 150
-		Stats: Armor, Sight, Speed, Damage, ReloadDelay, MaxRange
+		Stats: Armor, Sight, Speed, Damage, ReloadDelay, MaxRange, MinRange
```

Comment:
What is this for? Tomahawk can't attack air. MinRange stat means weapon with the least range, not Minimum Range of the weapon, that one is always shown.

Reply:
oh, that is a test line on our ActorStatValues.

Reply:
Removed.